examples/1_Introduction/05_usb_limited_angle_fbp_sirt.ipynb

```diff
@@ -24,7 +24,7 @@
     "#   See the License for the specific language governing permissions and\n",
     "#   limitations under the License.\n",
     "\n",
-    "#   Copyright 2021 UKRI-STFC, Technical University of Denmark\n",
+    "#   Copyright 2022 UKRI-STFC, Technical University of Denmark\n",
```

Comment:
should it be a range of years?

Reply:
Yes 2021-2022 I guess since I wrote it for the fully3d.